dipy/workflows/tests/test_io.py

```diff
@@ -68,7 +68,7 @@ def test_io_fetch_fetcher_datanames():
                      'DiB_217_lte_pte_ste', 'DiB_70_lte_pte_ste',
                      'synb0_weights', 'synb0_test', 'bundle_warp_dataset',
                      'evac_weights', 'evac_test', 'ptt_minimal_dataset',
-                     'stanford_tracks']
+                     'stanford_tracks', 'cti_rat1']
```

Comment:
This might fails because the name should be `cti_rat_data`. (same name as the fetcher - `fetch_cti_rat_data` since it reads the file)

Reply:
Hi @RafaelNH, see my comment above

Reply:
ok! I've changed everything to 'cti_rat1' including the name of the fetcher "fetch_cti_rat1". The name "fetch_cti_rat_data" may be inadequate if in future we want to add more or even improved rat cti datasets.